### PR TITLE
Fix pandaren action bars

### PR DESCRIPTION
**Pull request by jasongdove**
This configures Pandaren action bars for all classes.

It also casts `Stance of the Fierce Tiger` https://www.wowhead.com/spell=103985/stance-of-the-fierce-tiger on Pandaren Monk player create, and fixes the core so this stance is not removed on logout.

### CODE REVIEW

File: src/server/game/Server/WorldSession.cpp [hunk 1]
```diff
@@ -553,9 +553,6 @@ void WorldSession::LogoutPlayer(bool save)
             _player->SetPendingBind(0, 0);
         }
 
-        if (_player->HasAuraType(SPELL_AURA_MOD_SPEED_ALWAYS))
```
Comment (jasongdove) [inline src/server/game/Server/WorldSession.cpp:556]: I don't see this in any other cores, and it removes `Stance of the Fierce Tiger` which should not happen.

File: src/server/game/Entities/Player/Player.cpp [hunk 2]
```diff
@@ -661,15 +661,19 @@ bool Player::Create(uint32 guidlow, CharacterCreateInfo* createInfo)
     // original spells
     LearnDefaultSkills();
 
-    if (GetClass() == CLASS_MONK)
-        CastSpell(this, 103985, true); // Stance of the Fierce Tiger
```
Comment (jasongdove) [inline src/server/game/Entities/Player/Player.cpp:665]: This was moved to the new `playercreateinfo_cast_spell` table.

**Comment by leelf00** [comment]
Is this PR related to https://github.com/Legends-of-Azeroth/Legends-of-Azeroth-Pandaria-5.4.8/pull/175 for new tables?

**Comment by jasongdove** [comment]
Ah, I didn't see that PR before. This PR also adds `Quaking Palm` to all Pandaren class action bars, so #175 would be closed if this is merged.

**Comment by leelf00** [comment]
Maybe also relate to https://github.com/Legends-of-Azeroth/Legends-of-Azeroth-Pandaria-5.4.8/pull/174,https://github.com/Legends-of-Azeroth/Legends-of-Azeroth-Pandaria-5.4.8/pull/168,https://github.com/Legends-of-Azeroth/Legends-of-Azeroth-Pandaria-5.4.8/pull/165

**Comment by jasongdove** [comment]
Yes, I can follow up by configuring all race/class action bars if that would help.

I don't think the `playercreateinfo_spell` table is needed. It's not in TC/TCPP and the spells are already learned in this core.

**Comment by leelf00** [comment]
TCPP only have table `playercreateinfo_spell_custom`.

**Comment by jasongdove** [comment]
The table `playercreateinfo_spell_custom` is only used for config `PlayerStart.AllSpells = 1`. Spells for player creation are now loaded from DBC, not `playercreateinfo_spell`.

**Pull request merged**